src/components/rockets/rockets.js

```diff
@@ -1,21 +1,31 @@
 import React from 'react';
-import logoImage from '../../images/planet.png';
+import PropTypes from 'prop-types';
 import './rockets.css';
 
-export default function RocketCard() {
+const RocketCard = (props) => {
```

Comment:
- [ ] Do Style the Rocket card as per the design as per the milestone requirements, add some `margin` or `padding` between the card contents.
- [ ] The `description` also covers the entire available screen size width, please add a max-width.
_Take a look at the screenshots below_

### Required design

![microverse_rocket](https://user-images.githubusercontent.com/47344665/162024956-1a478885-2f42-4cdd-ac0d-9c5a4d46f08e.png)

### Your design

![emmuel](https://user-images.githubusercontent.com/47344665/162025038-5b315b2a-c71d-43b5-88c6-c84e271213a5.png)

### Project requirement

![design_rule](https://user-images.githubusercontent.com/47344665/162025102-06d161fb-5688-48d5-b95a-5822e48ffd48.png)

Reply:
This was done in another PR